src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -21,7 +21,8 @@ namespace ray {
 
 ClusterResourceScheduler::ClusterResourceScheduler(
     int64_t local_node_id, const NodeResources &local_node_resources)
-    : local_node_id_(local_node_id) {
+    : local_node_id_(local_node_id),
+      gen_(std::chrono::high_resolution_clock::now().time_since_epoch().count()) {
```

Comment:
I also think that we need a better seeding strategy to prevent biased samplings than using just a single 32-bit "nanoseconds from the epoch" integer, but that should probably be done in a PR that creates a utility and updates all such seedings.

Reply:
Yeah this is the same as other parts of code, so let's just keep it in this way & fix everywhere all together.